GE_Requirements.mediawiki

```diff
@@ -50,56 +64,62 @@ This project is part of [FIWARE](http://fiware.org/) and has been rated as follo
 standard [mailto:fiware-help@lists.fiware.org FIWARE Help Desk channel] and QA testings executed by QA Lab team.
 
 The following assessments will be conducted:
 
 * Assessment on support, to be performed by automated monitoring tools defined by the FIWARE Foundation.
-* Assessment on functionality, (sw and documentation), non-functional requirements, etc. to be performed by the QA Lab team.
+* Assessment on functionality, (sw and documentation), non-functional requirements, etc. to be performed by the FIWARE QA Lab team.
 
-Results of assessments lead to QA label assignment based on [https://www.fiware.org/wp-content/uploads/2016/10/QA_public_document.pdf FIWARE QA labeling schema].
-QA labels of a given FIWARE GE will always be associated to a minor release of FIWARE. In the event of not meeting minimal QA requirements:
+Results of these assessments lead to the assignment of a QA label based on [https://www.fiware.org/wp-content/uploads/2016/10/QA_public_document.pdf FIWARE QA labeling schema].
+QA labels of a given FIWARE GE will always be associated to a minor release of FIWARE. In the event of not meeting the minimal QA requirements:
 
 * ''Yellow card'' reprimand will be issued when overall QA label is '''B''' or lower.
 * ''Red card'' reprimand when overall QA label is '''C''' or lower or keeps being '''B''' in two consecutives minor releases of FIWARE.
 
-== Licensing Requirements ==
+== Licensing and IPR Management Requirements ==
+
+The source code of the product MUST be licensed under [https://opensource.org/licenses/alphabetical one of the well-recognized open source licenses approved by the Open Source Initiative].
+The open source license under which source code of the product is licensed MUST be clearly mentioned in a first-level section of the README.md file
+included in the main GitHub repository. When using a copyleft open source license, the following paragraph '''MUST''' be added in the section
+where the open source license is mentioned:
 
-When using a copyleft open source license IPR Management rules have to be clearly stated:
+::
+Please note that software derived as a result of modifying the source code of the <name-of-product>
+software in order to fix a bug or incorporate enhancements IS considered a derivative work of the product.
+Software that merely uses or aggregates (i.e. links to) an otherwise unmodified version of existing software IS NOT considered a derivative work.
+::
+
+
+Contributions from third parties MUST be accepted. There MUST be a document clearly describing the terms under which the IPR of contributions to the source code
+of the product will be managed. Such document MUST be made accessible in (or map to) a first-level section of the README.md file included
+in the associated GitHub repositories. When using a copyleft open source license, IPR Management rules for contributions MUST include clauses securing thathave to be clearly stated:
 
```

Comment:
thathave -> that have

Reply:
fixed https://github.com/Fiware/developmentGuidelines/pull/10/commits/91f106d190416886f573c0f70c5c9182d6ad8865